include/notcurses/direct.h

```diff
@@ -7,6 +7,10 @@
 extern "C" {
 #endif
 
+#ifdef NOTCURSES_FFI
```

Comment:
could we not just do this once?

also, we can't just redefine `static` everywhere; it also applies to e.g. function static variables. i'd rather introduce `APIINLINE`, resolving to `API inline` when appropriate and `static inline` elsewhere. then everywhere that we have a public `static inline`, we'll just replace that with `APIINLINE`. make sense?

Reply:
The reason this is in all three header files is that we don't want to do the `static -> `API` change inside headers that these headers themselves include.